src/bundle/Resources/public/js/scripts/helpers/request.helper.js

```diff
@@ -0,0 +1,58 @@
+(function(global, doc) {
+    const eZ = (global.eZ = global.eZ || {});
+
+    /**
+     * Handles request error
+     *
+     * @function handleRequestError
+     * @param {Response} response
+     * @returns {Error|Promise}
+     */
+    const handleRequestError = (response) => {
```

Comment:
`handleRequest` because it handles request. If there's an error then it throws an error, otherwise it returns a response.

Reply:
What do you think about `handleResponseError`?

Reply:
No because it does 2 things. Handling the error case is one of them. It just handles a response from request.